provider-directory/src/main/java/gov/va/api/health/providerdirectory/service/controller/practitioner/PractitionerTransformer.java

```diff
@@ -1,3 +1,130 @@
 package gov.va.api.health.providerdirectory.service.controller.practitioner;
 
-public class PractitionerTransformer {}
+import static gov.va.api.health.providerdirectory.service.controller.Transformers.allBlank;
+import static gov.va.api.health.providerdirectory.service.controller.Transformers.convert;
+
+import gov.va.api.health.providerdirectory.api.datatypes.Address;
+import gov.va.api.health.providerdirectory.api.datatypes.ContactPoint;
+import gov.va.api.health.providerdirectory.api.resources.Practitioner;
+import gov.va.api.health.providerdirectory.api.resources.Practitioner.Gender;
+import gov.va.api.health.providerdirectory.api.resources.Practitioner.PractitionerHumanName;
+import gov.va.api.health.providerdirectory.api.resources.Practitioner.PractitionerIdentifier;
+import gov.va.api.health.providerdirectory.service.ProviderContacts;
+import gov.va.api.health.providerdirectory.service.ProviderResponse;
+import gov.va.api.health.providerdirectory.service.ProviderWrapper;
+import gov.va.api.health.providerdirectory.service.controller.EnumSearcher;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class PractitionerTransformer implements PractitionerController.Transformer {
+
+  Boolean active(String active) {
+    if (active.toLowerCase().equals("active")) {
+      return true;
+    }
+    return false;
+  }
+
+  Address address(ProviderResponse.Value value) {
+    return convert(
+        value,
+        ppms ->
+            Address.builder()
+                .city(ppms.addressCity())
+                .country(ppms.addressCountry())
+                .state(ppms.addressStateProvince())
+                .build());
+  }
+
+  List<Address> addresses(ProviderResponse.Value value) {
+    if (value.address() == null) {
+      return null;
+    }
+    List<Address> addressList = new ArrayList<>();
+    addressList.add(address(value));
+    return addressList;
```

Comment:
This method combined with the one above is confusing. We check the value of `address`, but that's not what is actually used to build up the address. And then we should not build the address if city, country, and province are all blank. The methods should be combined together into something like this:

```
  List<Address> addresses(ProviderResponse.Value value) {
    if (value == null || allBlank(value.addressCity(), value.addressCountry(), value.addressStateProvince())) {
      return null;
    }
    return singletonList(
        Address.builder()
            .city(trimToNull(value.addressCity()))
            .country(trimToNull(value.addressCountry()))
            .state(trimToNull(value.addressStateProvince()))
            .build());
  }

```